packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2150,6 +2150,21 @@ export default class Meeting extends StatelessWebexPlugin {
     );
   };
 
+  /**
+  *  Checks if the machine has at least one audio or video device
+  * @param {object} [options]
+  * {
+  *    sendAudio: true/false,
+  *    sendVideo: true/false
+  * }
+  * @returns {Object} {
+  *    sendAudio: true/false,
+  *    sendVideo: true/false
+  *}
+  * @memberof Meetings
+  */
+ getSupportedDevice = (options) => Media.getSupportedDevice(options);
```

Comment:
If the options object is empty, could we default to `sendAudio: true, sendVideo:true`? That way you can just call `meeting.getSupportedDevice()` and find out if you have an audio or video device.

Reply:
done!